docs/Running-Mastodon/Production-guide.md

```diff
@@ -82,6 +82,47 @@ server {
 }
 ```
 
+## Apache
+
+Setting up Mastodon behind Apache is possible as well, although you will need to enable [mod_proxy_wstunnel](https://httpd.apache.org/docs/trunk/mod/mod_proxy_wstunnel.html) beforehand. The configuration is then pretty straightforward.
+
+```
+<VirtualHost *:80>
+   ServerAdmin contact@example.com
+   ServerName example.com
+   Redirect Permanent / https://example.com/
+</VirtualHost>
+
+<VirtualHost *:443>
+   ServerAdmin contact@example.com
+   ServerName example.com
+
+   DocumentRoot /home/mastodon/live/public/
+
+   Header add Strict-Transport-Security "max-age=31536000"
+   SSLEngine on
+   SSLProtocol All -SSLv2 -SSLv3
+   SSLHonorCipherOrder on
+   SSLCipherSuite EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH:AES128+EECDH:AES128+EDH
```

Comment:
You shoud disabled DHE ciphers : 
`SSLCipherSuite EECDH+AESGCM:AES256+EECDH:AES128+EECDH`